source/configure/smtp-email.rst

```diff
@@ -109,6 +109,11 @@ For example, if **System Console > Logs** displays the following error, search f
 
   Connection unsuccessful: Failed to add to email address - 554 5.7.1 <unknown[IP-ADDRESS]>: Client host rejected: Access denied
 
+Checking your Notifications settings
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+If SMTP sending fails with message ``Failed to set the to address``, go to **System Console > Site Configuration > Notifications** to verify that **Notification Display Name** and **Notification From Address** are set correctly. You may also need to define a **Support Email Address** and **Notification Reply-To Address**.
```

Comment:
>``Failed to set the to address``

Where would they expect to see this error? It is not clear to me if it will show in the logs or in product after an action?